### PR TITLE
[1.5] Update CODEOWNERS and master references

### PR DESCRIPTION
**Pull request by fpesce**

### CODE REVIEW

**Review by rlenglet — APPROVED** [review]

**Review by howardjohn — APPROVED** [review]

File: install/kubernetes/helm/istio-init/values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ global:
   hub: gcr.io/istio-testing
 
   # Default tag for Istio images.
-  tag: latest
```
Comment (howardjohn) [inline install/kubernetes/helm/istio-init/values.yaml]: indent broke here

File: install/kubernetes/helm/istio-init/values.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ global:
...
+tag: 1.5-dev
```
Comment (howardjohn) [inline install/kubernetes/helm/istio-init/values.yaml]:
```suggestion
  tag: 1.5-dev
```

**Review by dgn — APPROVED** [review]

**Pull request merged**